include/manager.hpp

```diff
@@ -244,6 +244,25 @@ class Manager
         const types::Path& i_fruPath, const nlohmann::json& i_sysCfgJsonObj,
         const types::WriteVpdParams i_paramsToWriteData);
 
+    /**
+     * @brief An API to get D-bus representation of given keyword name
+     *
+     * @param[in] i_keywordName - Keyword name.
+     *
+     * @return D-bus representation of keyword name.
+     */
+    inline std::string getDbusKeywordName(const std::string& i_keywordName)
```

Comment:
The function better suits as a utility, can be moved to vpd specific utility file.

Reply:
done